server/controllers/excelController.js

```diff
@@ -44,7 +47,100 @@ excelController.read = async (req, res, next) => {
 excelController.convertInputs = async (req, res, next) => {
   console.log('entering excelController.convertInputs');
   try {
+    const sample = { People: 'A', Species: 'I', Movies: 'Q' };
```

Comment:
Now that we also pass these in the request body, should we move these to `misc` as well?

Reply:
I was gonna keep this here until we do E2E testing.